Makefile

```diff
@@ -15,6 +15,7 @@ CONCORD_BFT_CONTAINER_CC?=clang
 CONCORD_BFT_CONTAINER_CXX?=clang++
 CONCORD_BFT_CMAKE_BUILD_TYPE?=Release
 CONCORD_BFT_CMAKE_BUILD_TESTING?=TRUE
+CONCORD_BFT_CLANG_TIDY?=/concord-bft/tools/run-clang-tidy.py
```

Comment:
Could you please build the path using the top-level variable `CONCORD_BFT_TARGET_SOURCE_PATH`?